proposals/kubeflow-distributions.md

```diff
@@ -0,0 +1,70 @@
+
+## Objective
+
+Clarify how Kubeflow distributions will be owned and developed going forward.
+
+## Motivation
+
+Kubeflow can be divided into pieces
+
+  1. Individual Kubeflow applications (e.g. Pipelines, KFServing, notebooks, etc...)
+  1. Distributions of Kubeflow (e.g. Kubeflow on GCP, Kubeflow on AWS, MiniKF, etc...)
+
+
+ Since July, the Kubeflow community has been working on forming working groups to create greater
+ accountability for the different parts of Kubeflow.
+
+ At this point in time, Kubeflow has formed working groups with clear ownership for all of the individual Kubeflow
+ applications.
+
+ There is an ongoing debate about who should own and maintain Kubeflow distributions.
+
+ To date there are two categories of distributions
+
+  1. Kubeflow distributions tied to a specific platform (e.g. AWS, GCP, etc...)
+  1. Generic distributions (e.g. for MiniKube, any conformant K8s cluster, etc...)
+
+The former have been owned and maintained by the respective vendors. The general consensus is that these should continue
+to be owned and maintained by the respective vendors outside any KF working group.
+
+This leaves the question of what to do about generic distributions. In particular, in [kubeflow/community#402](https://github.com/kubeflow/community/pull/402) there was a long debate about whether the deployments working group would own them or not. That discussion appears to be converging with the decision being that the deployments working group will not own any distributions.
+
+## Proposal
+
+Going forward all distributions of Kubeflow should be owned and maintained outside of Kubeflow. 
```

Comment:
All? I thought we were going to have a generic version owned by the deployment WG.

Reply:
Please review: kubeflow/community#402 distributions of Kubeflow are out of scope for the deployments WG.